packages/core/src/editing/design-library.ts

```diff
@@ -164,3 +186,20 @@ export function getDesignLibraryScriptLink(sitecoreEdgeUrl = SITECORE_EDGE_URL_D
 export function isDesignLibraryMode(mode: unknown): mode is DesignLibraryMode {
   return mode === DesignLibraryMode.Normal || mode === DesignLibraryMode.Metadata;
 }
+
+/**
+ * Sends an event to the Design Library
+ * @param {unknown} evt - The event object to send.
+ */
+export const postToDL = (evt: unknown) => {
```

Comment:
Let's follow our common naming convention for convenience
```suggestion
export const postToDesignLibrary = (evt: unknown) => {
```